src/payment/strategies/googlepay/googlepay-adyenv2-payment-processor.spec.ts

```diff
@@ -6,14 +6,14 @@ import { getCartState } from '../../../cart/carts.mock';
 import { createCheckoutStore, CheckoutRequestSender, CheckoutStore, CheckoutValidator } from '../../../checkout';
 import { getCheckoutState } from '../../../checkout/checkouts.mock';
 import { RequestError } from '../../../common/error/errors';
-import { getConfigState } from '../../../config/configs.mock';
+import { getConfig, getConfigState } from '../../../config/configs.mock';
 import { getCustomerState } from '../../../customer/customers.mock';
 import { OrderActionCreator } from '../../../order';
 import { createSpamProtection, PaymentHumanVerificationHandler } from '../../../spam-protection';
 import { getGooglePayAdyenV2, getPaymentMethodsState } from '../../payment-methods.mock';
 import { AdyenV2ScriptLoader } from '../adyenv2';
 
-import GooglePayAdyenV2PaymentProcessor from './googlepay-adyenv2-payment-processor';
```

Comment:
💅🏻  It would be better if you use index file `import { GooglePayAdyenV2PaymentProcessor } from './index';`

Reply:
the index file is mostly used to import things from other modules. If it's within the same module, it's preferred to use the filename